apps/external-devices/device-descriptors/Atmel_ATmega32.json

```diff
@@ -0,0 +1,13 @@
+{
+    "deviceVendor": "Atmel Corporation",
+    "deviceName": "ATmega32",
```

Comment:
@rpavlik Can we add a notes field for things like this? A lot of devices have caveats associated with them (e.g., device only works in this particular mode or device has variable number of analog/digital inputs that must be use-specified).  It'd also be nice to have a url field that could point to the manufacturer's product page.